src/modules/did/didSyncStatus.entity.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { DIDDocumentEntity } from './did.entity';
+import { DidSyncStatus } from './did.types';
+
+@Entity()
+export class DidSyncStatusEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @OneToOne(() => DIDDocumentEntity, (document) => document.id)
```

Comment:
Addinig relation only on DidSyncStatusEntity side. The DidDocumentEntity will not be changed